src/pybind/mgr/dashboard/frontend/src/app/core/layouts/workbench-layout/workbench-layout.component.ts

```diff
@@ -34,6 +34,6 @@ export class WorkbenchLayoutComponent implements OnInit, OnDestroy {
   }
 
   isDashboardPage() {
-    return this.router.url === '/dashboard';
+    return this.router.url === '/dashboard' || this.router.url === '/dashboard_3';
```

Comment:
If the above doesn't work you could try something [this alternative form](https://www.npmjs.com/package/ngx-pipe-function):
```
router.url | pipeFunction: Array.prototype.includes: ['/dashboard', '/dashboard_3']
```
That internally calls `Array.prototype.includes(['/dashboard', '/dashboard_3'], router.url)`. I'm not familiar with JS bound method internals, but perhaps bound methods have to be put in this way.

An alternative would be creating a specific pipe for the includes method `[1,2,3] | contains: 2`

Reply:
@epuertat I just can't make it work. Would appriciate if anyone else could also try it. @bryanmontalvan , @melissa-kun-li , could you check it out aswell?
And I'm pretty sure the `NgxPipeFunctionModule` is not globablly imported, so we would have to import it to `core.module.ts`, correct me if I'm wrong.

Reply:
You're right, the NgxPipeFunctionModule is imported 16 times, each one for a different component, and while I appreciate isolation for code, in the case of template helpers (like this) I feel it's better to make them widely available. We should definitely add that to the existing `shared/pipes.module.ts` so all our pipes are available from the shared module.

Reply:
Yeah, it would make sense, should I make a new tracker for it @epuertat ? Or is there no need since is a quick change and I can just open a PR for it?